packages/qb/pqb/src/sql/where.ts

```diff
@@ -205,64 +205,55 @@ const processWhere = (
           `${prefix}EXISTS (SELECT 1 FROM ${target} WHERE ${conditions} LIMIT 1)`,
         );
       }
-    } else if (
-      typeof value === 'object' &&
-      value &&
-      value.constructor === Object
-    ) {
-      if (isRaw(value)) {
-        ands.push(
-          `${prefix}${revealColumnToSql(
-            query,
-            query.shape,
-            key,
-            quotedAs,
-          )} = ${getRaw(value, ctx.values)}`,
-        );
-      } else {
-        let column = query.shape[key];
-        let quotedColumn: string | undefined;
-        if (column) {
-          quotedColumn = qc(column.data.name || key, quotedAs);
-        } else if (!column) {
-          const index = key.indexOf('.');
-          if (index !== -1) {
-            const joinedTable = key.slice(0, index);
-            const joinedColumn = key.slice(index + 1);
-            column = query.joinedShapes?.[joinedTable]?.[
-              joinedColumn
-            ] as typeof column;
-            quotedColumn = qc(
-              column?.data.name || joinedColumn,
-              q(joinedTable),
-            );
-          } else {
-            quotedColumn = undefined;
-          }
+    } else if (typeof value === 'object' && value && isRaw(value)) {
+      ands.push(
+        `${prefix}${revealColumnToSql(
+          query,
+          query.shape,
+          key,
+          quotedAs,
+        )} = ${getRaw(value, ctx.values)}`,
+      );
+    } else if (typeof value === 'object' && value?.constructor === Object) {
```

Comment:
`value.constructor === Object` is here to skip `Date` values, I see it won't no longer pass for the raw objects.

Somehow, we'll need to restructure this to not duplicate `typeof value === 'object'` in two `if`s.